magnetis/astro-native
# chore(react-native): update react native to version 0.68.7 in astro-native [CONFIA-1383]

# What


Update react native to version 0.68.7 in astro-native
# Why


To keep the project with more up-to-date features and support.
# How



# Sample


[N/A]
# QA

1. Pull the branch and do a `yarn` and `cd ios && pod install`;
2. Run storybook with `yarn storybook`;
3. Run it on iOS and Android emulators;
4. See if everything is working as it used to be.








[CONFIA-1383](https://produtomagnetis.atlassian.net/browse/CONFIA-1383)

[CONFIA-1383]: https://produtomagnetis.atlassian.net/browse/CONFIA-1383?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
